runtime/doc/support.txt

```diff
@@ -28,10 +28,10 @@ your Windows version, run the "winver" command and look for "Version xxxx"
 Support types ~
 
 * Tier 1: Officially supported and tested with CI. Any contributed patch
```

Comment:
What is the definition of "support"?

Reply:
"Guaranteed to work", alternatively "tested with CI" IMO.